cmd/soci/commands/index/list.go

```diff
@@ -103,10 +103,11 @@ var listCommand = cli.Command{
 
 func writeArtifactEntry(w io.Writer, ae *soci.ArtifactEntry, imageRef string) {
 	w.Write([]byte(fmt.Sprintf(
-		"%s\t%d\t%s\t%s\t\n",
+		"%s\t%d\t%s\t%s\t%v\t\n",
 		ae.Digest,
 		ae.Size,
 		imageRef,
 		ae.Platform,
+		ae.MediaType == soci.ORASManifestMediaType,
```

Comment:
Is it an ORASManifestMediaType even for the Referrers API?

Reply:
Yes.